day_6/exercises/burrito.rb

```diff
@@ -5,15 +5,33 @@
 # 3. change_protein
 
 class Burrito
-  attr_reader :protein, :base, :toppings
+  attr_accessor :protein, :base, :toppings
   def initialize(protein, base, toppings)
     @protein  = protein
     @base     = base
     @toppings = toppings
   end
+
+  def add_topping(toppings)
+    @toppings = toppings
+  end
+
+  def remove_topping
+    toppings.delete! toppings
```

Comment:
Similarly, here we'd like to see you pass in a specific topping to remove, and then use an array method that deletes that topping from the `@toppings` array. Can you find one that can do that?

Reply:
I couldn't find any documentation in the Day_6 lesson that speaks to what you're asking. I also couldn't find the info online or get any implementations to work. I'm going to be out of cell range and unable to process this further until late Sunday night. If you had gotten this to me sooner I would have more time to struggle through it.